vite.config.ts

```diff
@@ -1,5 +1,5 @@
 import { sveltekit } from "@sveltejs/kit/vite";
-import { defineConfig } from "vite";
+import { defineConfig } from "vitest/config";
```

Comment:
Is it ok to do this?

Reply:
I think it's ok but we actually don't need this change since we are using only default config! I'll revert.